src/corehost/cli/json/casablanca/include/cpprest/json.h

```diff
@@ -1293,12 +1293,13 @@ namespace json
         /// <returns><c>true</c> if the number fits into unsigned int64, <c>false</c> otherwise</returns>
         bool is_uint64() const
         {
+            assert(m_type == signed_type || m_type == unsigned_type || m_type == double_type);
```

Comment:
I'm not sure about this one. Assert is debug only, so on release builds... what will the function return? Can we please keep the default case in the switch?

Reply:
Clang points out that the switch cases already cover all the possible values. `default` will not be triggered unless additional enum types are added to `m_type`. If/when that happens, the assert should kick in, as should separate warnings from the compiler.

I am going to revert this commit anyway.

Reply:
Reverted.